training/src/anemoi/training/train/forecaster.py

```diff
@@ -98,7 +98,18 @@ def __init__(
 
         self.logger_enabled = config.diagnostics.log.wandb.enabled or config.diagnostics.log.mlflow.enabled
 
-        variable_scaling = self.get_variable_scaling(config, data_indices)
+        variable_scaling = GeneralVariableLossScaler(
+            config.training.variable_loss_scaling,
+            data_indices,
+        ).get_variable_scaling()
+
+        # Instantiate the pressure level scaling class with the training configuration
+        pressurelevelscaler = instantiate(
+            config.training.pressure_level_scaler,
```

Comment:
I think this config location is wrong? It should be training.variable_loss_scaling.pressure_level_scalar

Reply:
Indeed, in the config file, the pressure_level_scalar should be defined directly in training as before.